ocaml/idl/datamodel.ml

```diff
@@ -18,7 +18,7 @@ open Datamodel_types
 (* IMPORTANT: Please bump schema vsn if you change/add/remove a _field_.
               You do not have to bump vsn if you change/add/remove a message *)
 let schema_major_vsn = 5
-let schema_minor_vsn = 109
+let schema_minor_vsn = 110
```

Comment:
which field was removed/added?

Reply:
I don't think we should change the version number in a backport unless we really have to. It might make upgrades to next release more difficult (because they would be partially upgrading, partially downgrading the database when installing a new RTM build).
This might've been bumped to trigger the update code, but I see I used `fun _ -> true` in the upgrade code, so the version number is ignored. I think it'd be safe to not bump the version number here, as long as the upgrade code is still triggered.

Reply:
This might be a mistake. In on of the original commits the schema was updated but this might not be required for the backport.